docs/variants/minnowboard-turbot/test-matrix.md

```diff
@@ -0,0 +1,79 @@
+| 11. | [UEFI Shell][USH]                     | USH           | All                                  |
+| 12. | [USB detection][UDT]                  | UDT           | All                                  |
+| 13. | [USB booting][UBT]                    | UBT           | All                                  |
+| 14. | [Dasharo Tools Suite][DTS]            | DTS           | DTS001.001, DTS002.001, DTS003.001, DTS004.001, DTS005.001, DTS006.001 |
+| 15. | [CPU status][CPU]                     | CPU           | CPU001.001, CPU001.002               |
+
+[EFI]: ../../unified-test-documentation/dasharo-compatibility/30M-uefi-compatible-interface.md
+[DSP]: ../../unified-test-documentation/dasharo-compatibility/31E-display-ports-and-lcd.md
+[NBT]: ../../unified-test-documentation/dasharo-compatibility/315b-netboot-utilities.md
+[CLG]: ../../unified-test-documentation/dasharo-compatibility/304-custom-logo.md
+[CBK]: ../../unified-test-documentation/dasharo-compatibility/303-custom-boot-menu-key.md
+[USB]: ../../unified-test-documentation/dasharo-compatibility/306-usb-hid-and-msc-support.md
+[LBT]: ../../unified-test-documentation/dasharo-compatibility/308-debian-stable-and-ubuntu-lts-support.md
+[PXE]: ../../unified-test-documentation/dasharo-compatibility/315-network-boot.md
+[CNB]: ../../unified-test-documentation/dasharo-compatibility/30A-custom-network-boot-entries.md
+[USH]: ../../unified-test-documentation/dasharo-compatibility/30P-uefi-shell.md
+[DMI]: ../../unified-test-documentation/dasharo-compatibility/31L-smbios.md
+[AUD]: ../../unified-test-documentation/dasharo-compatibility/31F-audio-subsystem.md
+[UDT]: ../../unified-test-documentation/dasharo-compatibility/31O-usb-detect.md
+[UBT]: ../../unified-test-documentation/dasharo-compatibility/31N-usb-boot.md
+[CPU]: ../../unified-test-documentation/dasharo-compatibility/31T-cpu-status.md
+[DTS]: ../../unified-test-documentation/dasharo-compatibility/326-dasharo-tools-suite.md
+
+## Module: Dasharo security
+
+| No. | Supported test suite                  | Test suite ID | Supported test cases                 |
+|:----|:--------------------------------------|:-------------:|:-------------------------------------|
+| 1.  | [Secure Boot support][SBO]            | SBO           | All                                  |
+| 2.  | [BIOS lock support][BLS]              | BLS           | All                                  |
+| 3.  | [SMM BIOS write protection][SMM]      | SMM           | All                                  |
+| 4.  | [UEFI Setup password][PSW]            | PSW           | All                                  |
+| 5.  | [Network stack enable/disable][NBA]   | NBA           | All                                  |
+| 6.  | [USB stack enable/disable][USS]       | USS           | All                                  |
+
+[TPM]: ../../unified-test-documentation/dasharo-security/200-tpm-support.md
```

Comment:
I know this won't be visible on the result website, but it's probably best to remove the link since it might lead people to believe we support TPM already

Reply:
```suggestion
```

My miss.